semgrep-core/src/analyzing/Dataflow_constness.ml

```diff
@@ -280,22 +280,27 @@ and eval_concat env args =
 (* Transfer *)
 (*****************************************************************************)
 
-(* FIXME: This takes "Bottom" as the default constness of a variable.
```

Comment:
I would have removed the FIXME but kept most of the comment, especially the example.

Reply:
Good point, will do.